docs/asciidoc/alignments/alignments_output.adoc

```diff
@@ -37,3 +37,23 @@ the genomic template)
 |3_p_align_offset |Offset of the last nucleotide of the SW alignemnt
 (relative to the read)
 |=======================================================================
+
+
+
+The CDR3 files are semicolon separated files. For each IGoR indexed 
+sequence the following fields are given:
+[width="99%",cols="<32%,<68%",options="header",]
+|=======================================================================
+|Field |Description
+|seq_index |The sequence index the alignment corresponds to in the
+_indexed_sequences.csv_ file.
+
+|v_anchor | Position of the V anchor relative to the read sequence. 
```

Comment:
Maybe add a precise statement on index construction, e.g: "Index is 0 based and refers to the first letter of the V anchor's codon on the read"